tests/test_other.py

```diff
@@ -7986,9 +7986,9 @@ def run(*args):
       run(['-Os'],  0, [], [],          85,  0,  2,  2) # noqa
       run(['-Oz'],  0, [], [],          54,  0,  1,  1) # noqa
     else:
-      run([],      21, ['abort'], ['waka'], 22712, 22, 15, 30) # noqa
-      run(['-O1'], 10, ['abort'], ['waka'], 10450,  7, 11, 11) # noqa
-      run(['-O2'], 10, ['abort'], ['waka'], 10440,  7, 11, 11) # noqa
+      run([],      23, ['__runtime_abort'], ['waka'], 22712, 22, 15, 30) # noqa
+      run(['-O1'], 12, ['__runtime_abort'], ['waka'], 10450,  7, 11, 11) # noqa
+      run(['-O2'], 12, ['__runtime_abort'], ['waka'], 10440,  7, 11, 11) # noqa
```

Comment:
what are the changes here caused by?

Reply:
the increased in sending is caused by forcing getTempRet0/setTempRet0 library functions, the "__runtime_abort" changes are due to the presence of the runtime version of abort being expected to exist in this test.

Reply:
I see, makes sense. This is a little annoying that we increase it for all programs, but it does go away in -O3/-Os thanks to metadce, so maybe it's worth it for the simplicity. I'm hoping we can see that simplificy when the other setTempRet* stuff is removed.

Reply:
Yeah there will be no more imports of the runtime versions of abort/setTempRet0/getTempRet0 (I will add that change tomorrow), so everything should be play nice with all existing optimizations and there should be parity with how the WASM backend imports those functions.